rtcp/src/source_description/mod.rs

```diff
@@ -22,8 +22,10 @@ const SDES_TEXT_OFFSET: usize = 2;
 /// RTP SDES item types registered with IANA. See: https://www.iana.org/assignments/rtp-parameters/rtp-parameters.xhtml#rtp-parameters-5
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 #[repr(u8)]
+#[derive(Default)]
 pub enum SdesType {
-    SdesEnd = 0,      // end of SDES list                RFC 3550, 6.5
+    #[default]
+    SdesEnd = 0, // end of SDES list                RFC 3550, 6.5
```

Comment:
Whitespace before comment should be consistent with the other lines.

Reply:
Fixed by https://github.com/webrtc-rs/webrtc/pull/435/commits/3df18edba87a0f7a69dbdc576d24e673994cdc21